src/app/components/add-card/add-card.component.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  Component,
+  OnInit,
+  ElementRef,
+  ViewChild,
+  Input,
+  Output,
+  EventEmitter,
+} from '@angular/core';
+import { formatDate } from '@angular/common';
+import * as DOMPurify from 'dompurify';
+
+import { UtilityService } from 'src/app/utility.service';
```

Comment:
Instead of putting the service randomly, how about we use a dedicated directory?